src/components/templates/Search/index.tsx

```diff
@@ -80,18 +80,12 @@ export default function SearchPage({
           <SearchBar initialValue={(text || owner) as string} />
         )}
         <div className={styles.row}>
-          <PriceFilter
-            priceType={price}
-            serviceType={service}
-            setPriceType={setPriceType}
-            setServiceType={setServiceType}
-          />
+          <PriceFilter serviceType={service} setServiceType={setServiceType} />
```

Comment:
This component should be renamed, maybe to `ServiceFilter` now that we removed the price. Also the name of the file containing the component

Reply:
Right! on it